.gitignore

```diff
@@ -19,4 +19,7 @@
 ###< symfony/phpunit-bridge ###
 ###> symfony/asset-mapper ###
 /public/assets/
+/assets/vendor/
 ###< symfony/asset-mapper ###
+
+messages.db
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏This should be removed now 😄